package.json

```
@@ -31,6 +31,7 @@
     "./package.json": "./package.json"
   },
   "dependencies": {
+    "@digitalbazaar/credentials-v2-context": "github:digitalbazaar/credentials-v2-context",
```

```suggestion
    "@digitalbazaar/credentials-v2-context": "digitalbazaar/credentials-v2-context",
```
You can use the shortened GH syntax here (no `github:` prefix necessary)